Include/internal/pycore_stackref.h

```diff
@@ -191,18 +191,15 @@ PyStackRef_FromPyObjectImmortal(PyObject *obj)
     } while (0)
 
 #ifdef Py_GIL_DISABLED
-static inline void
-PyStackRef_CLOSE(_PyStackRef stackref)
-{
-    if (PyStackRef_IsDeferred(stackref)) {
-        // No assert for being immortal or deferred here.
-        // The GC unsets deferred objects right before clearing.
-        return;
-    }
-    Py_DECREF(PyStackRef_AsPyObjectBorrow(stackref));
-}
+#   define PyStackRef_CLOSE(stackref) \
+        do { \
+            _PyStackRef _close_tmp = (stackref); \
+            if (!PyStackRef_IsDeferred(_close_tmp)) { \
+                Py_DECREF(PyStackRef_AsPyObjectBorrow(_close_tmp)); \
+            } \
+        } while (0)
```

Comment:
PEP 7 requires to:

- use uppercase letters for macro parameters,
- align line continuations `\`